system_test/helpers/aest_docker_api.erl

```diff
@@ -117,14 +117,12 @@ start_container(ID) ->
 
 stop_container(ID, Opts) ->
     STDefault = ?CONTAINER_STOP_TIMEOUT,
-    {Query, HTDefault} = case maps:get(soft_timeout, Opts, STDefault) of
+    {Query, ReqTimeout} = case maps:get(soft_timeout, Opts, STDefault) of
         infinity -> {#{t => ?CONTAINER_STOP_INFINTY}, infinity};
-        STSecs ->
-            {#{t => STSecs}, STSecs + ?CONTAINER_KILL_TIMEOUT_EXTRA}
-    end,
-    ReqTimeout = case maps:get(hard_timeout, Opts, HTDefault) of
```

Comment:
As hard_timeout now unused it shall be removed from https://github.com/aeternity/epoch/pull/1060/files#diff-e2b7668405d6906af2515a075fe330c1R93 ?

Reply:
Good catch, it is not that hard_timeout is not used anymore, it was a something wrong in the PR.